src/deploy-output.ts

```diff
@@ -0,0 +1,22 @@
+import * as core from '@actions/core'
+import { getCliClient, getEnvironmentName } from './utils'
+
+export async function deployOutput(): Promise<void> {
```

Comment:
How about calling this function something like `outputEnvironmentUrl` so that it's clearer that it deals with the URL?

Reply:
Thanks for pointing this out. I have renamed it.